<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window_Bus</class>
 <widget class="QMainWindow" name="Window_Bus">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Bus</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../pics/hops_icon.PNG</normaloff>../pics/hops_icon.PNG</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_Initialization">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>330</width>
      <height>220</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Initialization</string>
    </property>
    <widget class="QLabel" name="label_FileName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Output file name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_FileName">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>190</width>
       <height>20</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>Write output file name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_BusNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bus number:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbox_BusNumber">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>190</width>
       <height>20</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_AddChannels">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Add channels:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkbox_Frequency">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Frequency</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkbox_Voltage">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>115</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Voltage</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkbox_VoltageAngle">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Voltage and angle</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Initialize">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Initialize</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Run">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>330</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Run Dynamics</string>
    </property>
    <widget class="QLabel" name="label_PauseTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Time of pause:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_PauseTime">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>190</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Run">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>65</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
     <property name="shortcut">
      <string>F5</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_RunTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>115</y>
       <width>110</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Run time: 0.0s</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_LineFault">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>330</width>
      <height>220</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Add disturbace - Line fault</string>
    </property>
    <widget class="QLabel" name="label_lf_FromNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>From Bus Number:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_lf_CircuitId">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Circuit ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_lf_ToNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>To bus number:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_lf_FaultTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>End time of fault:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbox_Lf_FromNumber">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>190</width>
       <height>20</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbox_Lf_ToNumber">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>190</width>
       <height>20</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_Lf_CircuitId">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>170</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_Lf_FaultTime">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>170</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Lf_AddDisturbance">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Add disturbance</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_BusFault">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>330</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Add disturbace - Bus fault</string>
    </property>
    <widget class="QLabel" name="label_bf_BusNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bus Number:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_bf_FaultTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>End time of fault:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbox_Bf_BusNumber">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>190</width>
       <height>20</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_Bf_FaultTime">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>170</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Bf_AddDisturbance">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Add disturbance</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_PlotGraph">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>405</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot graph</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+P</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Imported">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>405</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Imported files:
rawfile.raw
dyfile.dyr</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Status">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImport">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport_2">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="action_Raw"/>
     <addaction name="action_Sav"/>
     <addaction name="separator"/>
     <addaction name="action_Dyr"/>
     <addaction name="action_Snp"/>
    </widget>
    <addaction name="menuImport_2"/>
    <addaction name="action_PlotGraph"/>
   </widget>
   <widget class="QMenu" name="menuElement_Info">
    <property name="title">
     <string>Element Info</string>
    </property>
    <addaction name="action_Bus"/>
    <addaction name="action_Branch"/>
    <addaction name="action_Machine"/>
    <addaction name="separator"/>
    <addaction name="action_ShowGrid"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuImport"/>
   <addaction name="menuElement_Info"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Raw">
   <property name="text">
    <string>RAW</string>
   </property>
   <property name="statusTip">
    <string>Read a Power Flow Raw Data File</string>
   </property>
  </action>
  <action name="action_Sav">
   <property name="text">
    <string>Saved Case</string>
   </property>
   <property name="statusTip">
    <string>Read a Saved Case File</string>
   </property>
  </action>
  <action name="action_Dyr">
   <property name="text">
    <string>Dynamics Data File</string>
   </property>
   <property name="statusTip">
    <string>Read a Dynamics Data File</string>
   </property>
  </action>
  <action name="action_Snp">
   <property name="text">
    <string>Dynamics Snapshot File</string>
   </property>
   <property name="statusTip">
    <string>Read a dynamics Snapshot File</string>
   </property>
  </action>
  <action name="action_Bus">
   <property name="text">
    <string>Bus</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="action_Branch">
   <property name="text">
    <string>Branch</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="action_Machine">
   <property name="text">
    <string>Machine</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="action_PlotGraph">
   <property name="text">
    <string>Plot graph</string>
   </property>
  </action>
  <action name="action_ShowGrid">
   <property name="text">
    <string>Show Grid</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
